package.json

```diff
@@ -24,6 +24,7 @@
     "@react-spectrum/tabs": "^3.0.0-alpha.3",
     "@spectrum-icons/workflow": "^3.2.0",
     "@types/negotiator": "^0.6.1",
+    "cookies": "^0.8.0",
```

Comment:
I wonder if this is necessary to delete the cookie, or if it would be possible to just delete the session using [`session.destroy()`](https://www.npmjs.com/package/next-session#reqsessiondestroy). Did you look into that or is this just the solution you came up with?